### DISCUSSION TITLE
Example with Modifiers in JSON Sample File Request

### DISCUSSION THREAD

**Discussion by knh762002** [CMSgov/hospital-price-transparency#22] [Q&A] [answered]
Will you update the sample .JSON file with an example that includes a Modifier?  I am not seeing a correlation between the standard charges and modifiers in the schema.  Any guidance on how to implement codes with modifiers would be greatly appreciated.

Thank you!

**Comment by RidenourK** [comment]
@knh762002 Thank you for your interest, we are working on more examples and will notify you when they are posted.

**Reply by nmayle** [reply]
The JSON version of including modifiers for a code doesn't seem to have a way to include multiple modifiers that apply simultaneously.

Also, the JSON version of the representing modifiers only seems to have a text field that describes how the modifier adjusts the standard charge ("payer specific negotiated charge"); it doesn't have a field for the actual dollar value.  

This seems to not be parallel to how and what can be represented in the CSV format.  The CSV format has a field for plural "modifiers".
The CSV format also has a whole row include "standard_charge | negotiated_dollar" for a row with a modifier or set of modifiers to the code.

This seems like a deficiency in the JSON representation since in Transparency in Coverage when there is modifier or a set of modifiers, there is a separate negotiated rate.  I believe that the modifier(s) would also alter the gross charge, cash charge, etc.   A procedure that is only done on the left side ("LT") would be billed and reimbursed at a different "rate" from one that was done bilaterally.

**Reply by knh762002** [reply]
I was just going to write a similar response as @nmayle regarding the Modifiers in the JSON schema.  Modifiers, in my opinion, should be included in the standard_charges object like they were in Version 1 and 1.1.  Having modifiers encoded in this way would allow for different negotiated rates for multiple payers in the same code_information object.  Thoughts or comments welcome.

**EXAMPLE:**

 "standard_charge_information": [
  {  "description": "Ultrasound, pregnant uterus, real time with image documentation........",
      "code_information": [
          {
              "code": "555555555",
              "type": "CDM"
          }, 
          {
              "code": "76801",
              "type": "CPT"
          }
      ],
      "standard_charges": [
      {
              "setting": "both",
              "billing_class": "professional",
              "gross_charge": 121.90,
              "discounted_cash": 91.42,
              "minimum": 40.03,
              "maximum": 60.99,
          **"modifiers": [
              "26"
          ],**
          "payers_information": [
          {
              "payer_name": "Payer 1",
              "plan_name": "PPO",
              "negotiated_algorithm": "Conditional payment logic at the claim level including adjustments as specified in the payer agreements.",
              "estimated_amount": 60.99,
              "methodology": "other"
          },
          {
              "payer_name": "Payer 2",
              "plan_name": "PPO",
              "standard_charge_percentage": 70,
              "estimated_amount": 44.03,
              "methodology": "percent of total billed charges"                       
          }
          ]
      }
      ]
  }
]

**Comment by knh762002** [comment]
Thank you!!




On Wednesday, January 10, 2024, 3:26 PM, RidenourK ***@***.***> wrote:




@knh762002 Thank you for your interest, we are working on more examples and will notify you when they are posted.

**Comment by RidenourK** [comment, marked as answer]
@knh762002. Thank you for your patience. 

Please refer to the updated JSON examples which include a modifier example.
